src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -89,11 +94,21 @@
     <Compile Include="Internal\Reflection\Core\NonPortable\RuntimeType.cs" />
     <Compile Include="Internal\Reflection\Core\NonPortable\RuntimeTypeUnifier.cs" />
     <Compile Include="Internal\Reflection\Core\NonPortable\RuntimeTypeUnifier.Internals.cs" />
-    <Compile Include="Internal\Reflection\Extensibility\ExtensibleType.cs" />
-    <Compile Include="Internal\Runtime\Augments\WinRTInterop.cs" />
-    <Compile Include="Internal\Reflection\ExplicitScopeAttribute.cs" />
     <Compile Include="Internal\Reflection\Core\NonPortable\ReflectionCoreNonPortable.cs" />
     <Compile Include="Internal\Reflection\MetadataTransformedAttribute.cs" />
+    <Compile Include="Internal\Reflection\ExplicitScopeAttribute.cs" />
+    <Compile Include="Internal\Reflection\Tracing\ReflectionTrace.Public.cs" />
+    <Compile Include="Internal\Reflection\Tracing\ReflectionTrace.Public.Events.cs" />
+    <Compile Include="Internal\Reflection\Tracing\ReflectionTrace.Internal.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(IsProjectNLibrary)' != 'true'">
+    <Compile Include="Internal\Reflection\CoreRT\RuntimeType.cs" />
```

Comment:
It may be interesting to beef up this simple implementation into the real thing (while keeping it simple). It does not look right to have all the reflection-related hash tables and stuff in System.CoreLib.